Header header

int32 lat_behavior

int32 lon_behavior

float32 actual_desired_velocity

VehicleTraj[] forward_trajs

int32[] forward_behaviors

SurroundTraj[] surround_trajs

State state
